backend/src/modules/device/device.controller.ts

```diff
@@ -30,4 +32,18 @@ export class DeviceController {
       return err;
     }
   }
+
+  @Post('/MQTT/propogateToMobile')
```

Comment:
nit: endpoint should be called `simulateMQTTMessage`

Also, isn't it easier to just publish an MQTT message instead of calling this? I guess it doesn't matter since you already wrote this endpoint.

Reply:
okok ill change the endpoint name

I didn't wanna write code to connect to the MQTT server on the street view ui, and fetch api is built in since the street view ui is just serving a single html and js file.